data/human/variants.txt

```diff
@@ -1936,6 +1990,22 @@ ship "Hauler III" "Hauler III (Hai)"
 	turret "Chameleon Anti-Missile"
 	turret "Quad Blaster Turret"
 
+ship "Hauler III" "Hauler III (CCOR)"
+	outfits
+		"Heavy Anti-Missile Turret" 3
+		"Anti-Missile Turret"
+		"NT-200 Nucleovoltaic"
+		"nGVF-AA Fuel Cell"
```

Comment:
The NT-200 is a very modern piece of deep technology and feels a bit out of place here.
```suggestion
		"nGVF-DD Fuel Cell"
```

Reply:
Yeah, the Hauler variant is the one I have yet to get to, and I agree. Though, I'm still debating if it makes sense to be a Hauler III for the broken CCOR, and not something smaller like a Hauler 2 or 1 (similar to how rare the carrier variants of these are the larger they get for Pirates).